### DISCUSSION TITLE
page not loading

### DISCUSSION THREAD

**Discussion by Caridad23** [community/community#60858] [npm] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello, pretty new here to github and in the programing community. Please don't pick on me, lol. 
Every time I attempt to upload a repository nothing comes up on the page. There isn't a problem with the code that I know of, or any other problems I can tell, but its always just a blank page. Can someone please tell me what I'm doing wrong?

Link to my latest repository:
https://caridad23.github.io/dodAgeCalc/ 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by AbdrrahimDahmani** [comment]
Hi @Caridad23, welcome to the community,
Well, GitHub Pages serves content from the "gh-pages" branch by default. check that you have a branch named "gh-pages" in your repository, and the content you want to display is present on that branch. You might check this topic for more informations!

**Reply by Caridad23** [reply]
Thank you for responding, I have the main branch selected.

**Comment by OshekharO** [comment, marked as answer]
Well the issue is quite simple you need to rename DogCalc.html to index.html

PR: https://github.com/Caridad23/dogAgeCalcu/pull/1

**Reply by Caridad23** [reply]
Thank you

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Caridad23 ! 

We're happy you're here and are getting acquainted with GitHub. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you to our Pages category. 

Don't forget to mark the answer if someone shares a solution ✅ .

**Reply by Caridad23** [reply]
My bad, will do. Thank you.
